src/test/java/org/opensrp/web/rest/PractitionerResourceTest.java

```diff
@@ -210,6 +210,29 @@ public void testUpdateWithJsonSyntaxException() throws Exception {
         verifyNoMoreInteractions(practitionerService);
     }
 
+	@Test
+	public void testGetPractitionersByPractitionerRoleIdentifierAndCode() throws Exception {
+		List<Practitioner> expectedPractitioners = new ArrayList<>();
+
+		Practitioner expectedPractitioner = initTestPractitioner1();
+		expectedPractitioners.add(expectedPractitioner);
+
+		doReturn(expectedPractitioners).when(practitionerService).getAssignedPractitionersByIdentifierAndCode(anyString(), anyString());
```

Comment:
Mock this only if the **practitionerIdentifier** = **test** and **code** = **testCode**. This will ensure the way this method is called does not break/change incase new code modifies the params before calling the service

Reply:
Done